apps/sensenet/cypress/integration/saved-queries/saved-queries.spec.ts

```diff
@@ -4,34 +4,43 @@ describe('Saved queries', () => {
   before(() => {
     cy.login()
     cy.visit(pathWithQueryParams({ path: '/', newParams: { repoUrl: Cypress.env('repoUrl') } }))
+    cy.viewport(1000, 800)
   })
   it('saving a query should work properly.', () => {
-    const inputText = "+InFolder:'/Root/Content/IT/Document_Library'"
+    const inputText = 'Sample'
     cy.get('[data-test="drawer-menu-item-search"]').click()
     cy.get('a[title="New search"]').click()
     cy.get('[data-test="input-search"]').type(inputText)
-    cy.get('[data-test="table-cell-chicago"]').should('be.visible')
-    cy.get('[data-test="table-cell-munich"]').should('be.visible')
+
+    cy.get('[data-test="table-cell-sample-workspace"]').should('be.visible')
```

Comment:
expected results could be an array of string, and just iterate over it here and also after loading it back